docs/data-sources/gaussdb_opengauss_instances.md

```diff
@@ -0,0 +1,109 @@
+---
+subcategory: "GaussDB"
+---
+
+# huaweicloud_gaussdb_opengauss_instances
+
+Use this data source to get available HuaweiCloud gaussdb opengauss instances.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_gaussdb_opengauss_instances" "this" {
+  name = "gaussdb-instance"
+}
+```
+
+## Argument Reference
+
+* `region` - (Optional, String) The region in which to obtain the instance. If omitted, the provider-level region will
+  be used.
+
+* `name` - (Optional, String) Specifies the name of the instance.
+
+* `vpc_id` - (Optional, String) Specifies the VPC ID.
+
+* `subnet_id` - (Optional, String) Specifies the network ID of a subnet.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - Indicates the ID of the instances.
```

Comment:
What' the difference between this `id` and the `id` in instances block?

Reply:
id within this block is separated instance id, the id outside is the data source id which is hashcode of the list ids